README.md

```diff
@@ -28,3 +28,127 @@ After matching the slide filenames against the sample IDs from the clinical data
 Regions of interest can be extracted using the python script generate_rois.py. This script consumes a tab-delimited text file describing the whole-slide image files, ROI coordinates, desired size and magnification for extracted ROIs, and then generates a collection of ROI .png images. These images are transformed into a binary for model training and testing by the software described below.
 
 *Note: region extraction depends on the [OpenSlide](http://openslide.org/) library. This library can be installed locally, and is also available in the Docker image described below.*
+
+# Executables for training and testing models
+
+The Docker container provides executables for training SCNN/GSCNN models and for evaluating the accuracy of these models. Both of these executables consume:
+* ROI .png files
+* A .csv file describing the patient IDs, event times, censoring (0 for uncensored, 1 for right-censored), patient indexes (integer indexes assigned to each patient), and any genomic variables. **_Note that the columns for genomic data should be organized in a way that the first coloumns are allocated for those which will NOT get normalized and the latter ones allocated for those that have to get normnalized._** The default normalization method in our model is Z score normalization. An example file is provided in this repository.
```

Comment:
Can we just have people responsible for their own normalization?

Reply:
Fixed it inside the code.